homeassistant/components/buienradar/sensor.py

```diff
@@ -114,30 +114,35 @@
         name="Humidity",
         native_unit_of_measurement=PERCENTAGE,
         icon="mdi:water-percent",
+        state_class="measurement",
```

Comment:
I would suggest using the constants:

https://github.com/home-assistant/core/blob/2a05697c9171a2d96d3f52cc2089b8c341b32ac7/homeassistant/components/sensor/__init__.py#L99-L104

Reply:
Yup, that is the way to go 👍

Reply:
Using constants now.